pom.xml

```diff
@@ -3,7 +3,7 @@
     <groupId>org.littleshoot</groupId>
     <artifactId>littleproxy</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.3.3-VGS-SNAPSHOT</version>
+    <version>1.1.3.4-VGS-SNAPSHOT</version>
```

Comment:
this is new functionality. i think you should be bumping the minor version number